lib/publiccloud/azure.pm

```diff
@@ -150,6 +155,21 @@ sub calc_img_version {
     return "$build.$kiwi";
 }
 
+sub get_image_definition {
```

Comment:
```suggestion
sub get_image_definition_from_azure {
```

Reply:
You would rename a internal subroutine in `azure.pm` from `get_image_definition` to `get_image_definition_from_azure` ?

Reply:
Renamed to `generate_azure_image_definition` because now it's also used outside the `azure.pm` module.

Reply:
ok , agree

Reply:
BUT currently we have `generate_azure_image_definition` and `get_image_definition`  : 
1. "azure"  in `generate_azure_image_definition` should be removed with same reasoning , right ?
2.  We have 2 functions one is "getting" image definition and another "generating" it . :thinking:  I don't have something certain to suggest here but I can just state that naming is a bit confusing ...

Reply:
ok after reading function description I would say it should be something like `get_image_definition_from_gallery` , WDYT ?

Reply:
Since the gallery is the default and there are no other means/variants I would argue that unnecessarily verbose. Or do we expect there to be multiple variations of `get_image_definition` in the foreseeable future?

Reply:
ok